docker/Dockerfile

```diff
@@ -1,7 +1,3 @@
 FROM scratch
-ARG version
-ARG target=x86_64-unknown-linux-musl
-
-ADD https://github.com/intenthq/gitkv/releases/download/${version}/gitkv-${target} /gitkv
-
-ENTRYPOINT [ "/gitkv" ]
+ADD gitkv /gitkv
```

Comment:
We can't pull from GitHub releases because we haven't necessarily published it yet! So this just grabs it locally now instead — see `build.sh` below.